### PR TITLE
Update redfish_info.py

### PR DESCRIPTION
**Pull request by Rasdva3**
##### ISSUE TYPE
- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/modules/remote_management/redfish/redfish_info.py [hunk 1]
```diff
@@ -67,7 +67,9 @@
       username: "{{ username }}"
       password: "{{ password }}"
     register: result
-  - ansible.builtin.debug:
+    
+  - name: Print fetched information
+    ansible.builtin.debug:
```
Comment (Andersson007) [inline plugins/modules/remote_management/redfish/redfish_info.py:72]:
```suggestion

  - name: Print fetched information
    ansible.builtin.debug:
```

File: plugins/modules/remote_management/redfish/redfish_info.py [hunk 2]
```diff
@@ -78,7 +80,9 @@
...
```
Comment (Andersson007) [inline plugins/modules/remote_management/redfish/redfish_info.py:85]:
```suggestion

  - name: Print fetched information
    ansible.builtin.debug:
```

File: plugins/modules/remote_management/redfish/redfish_info.py [hunk 3]
```diff
@@ -108,7 +112,9 @@
       username: "{{ username }}"
       password: "{{ password }}"
     register: result
-  - ansible.builtin.debug:
+    
+  - name: Print fetched information
```
Comment (Andersson007) [inline plugins/modules/remote_management/redfish/redfish_info.py:116]:
```suggestion

  - name: Print fetched information
```

File: plugins/modules/remote_management/redfish/redfish_info.py [hunk 4]
```diff
@@ -130,7 +138,9 @@
...
```
Comment (Andersson007) [inline plugins/modules/remote_management/redfish/redfish_info.py:142]:
```suggestion

  - name: Print fetched information
```

File: plugins/modules/remote_management/redfish/redfish_info.py [hunk 5]
```diff
@@ -139,7 +149,9 @@
       username: "{{ username }}"
       password: "{{ password }}"
     register: result
-  - ansible.builtin.debug:
+
```
Comment (Andersson007) [inline plugins/modules/remote_management/redfish/redfish_info.py:152]:
```suggestion

```

**Comment by Andersson007** [comment]
Fixes https://github.com/ansible-collections/community.general/pull/3485

File: plugins/modules/remote_management/redfish/redfish_info.py [hunk 6]
```diff
@@ -119,7 +125,9 @@
...
```
Comment (Andersson007) [inline plugins/modules/remote_management/redfish/redfish_info.py:128]:
```suggestion

```

**Comment by Andersson007** [comment]
the PR should be fine now
cc @felixfontein there are unrelated failings in CI

File: plugins/modules/remote_management/redfish/redfish_info.py [hunk 7]
```diff
@@ -67,7 +67,9 @@
...
```
Comment (Andersson007) [inline plugins/modules/remote_management/redfish/redfish_info.py:70]:
```suggestion
```

File: plugins/modules/remote_management/redfish/redfish_info.py [hunk 8]
```diff
@@ -67,7 +67,8 @@
       username: "{{ username }}"
       password: "{{ password }}"
     register: result
-  - ansible.builtin.debug:
+  - name: Print fetched information
```
Comment (Andersson007) [inline plugins/modules/remote_management/redfish/redfish_info.py:70]:
```suggestion

    - name: Print fetched information
```

File: plugins/modules/remote_management/redfish/redfish_info.py [hunk 9]
```diff
@@ -67,7 +67,9 @@
       username: "{{ username }}"
       password: "{{ password }}"
     register: result
-  - ansible.builtin.debug:
+
+    - name: Print fetched information
```
Comment (Andersson007) [inline plugins/modules/remote_management/redfish/redfish_info.py:71]:
```suggestion
  - name: Print fetched information
```

**Review by Andersson007 — APPROVED** [review]
LGMT

**Comment by felixfontein** [comment]
@Rasdva3 thanks for improving the docs!
@Andersson007 thanks for reviewing!

**Comment by Andersson007** [comment]
@Rasdva3 thanks for the contribution!
@felixfontein thanks for reviewing!

**Pull request merged**